code/game/objects/items/stacks/sheets/miscellaneous/miscellaneous_mats.dm

```diff
@@ -147,5 +147,4 @@ GLOBAL_LIST_INIT(sandbag_recipes, list ( \
 	material_type = /datum/material/meat
 	material_modifier = 1 //None of that wussy stuff
 
-/obj/item/stack/sheet/meat/get_recipes()
-	return SSmaterials.rigid_stack_recipes
+/obj/item/stack/sheet/meat/get_recipes() //empty typepath. Leave it be.
```

Comment:
I guess you missed my answer..?
You need to return a recipe glob that's dedicated for this like all other recipes do.

Reply:
fixed in 1ce8eb7